api/src/paths/markdown/{markdownId}/index.ts

```diff
@@ -0,0 +1,121 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SYSTEM_ROLE } from '../../../constants/roles';
+import { getDBConnection } from '../../../database/db';
+import { authorizeRequestHandler } from '../../../request-handlers/security/authorization';
+import { MarkdownService } from '../../../services/markdown-service';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('paths/markdown/{markdownId}/index');
+
+export const POST: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          validSystemRoles: [SYSTEM_ROLE.PROJECT_CREATOR, SYSTEM_ROLE.SYSTEM_ADMIN, SYSTEM_ROLE.DATA_ADMINISTRATOR],
```

Comment:
I assume all users should be able to see these help records right?

I think we would want the auth to just be:
```
      {
        discriminator: 'SystemUser'
      }
```

So any logged in user can access the help and cast votes.

Reply:
I've updated it, can change it later if needed of course